CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Changelog 
 
+## 1.56.0
+
+- fix downscaling images #2469
+
+- fix outgoing messages popping up in selfchat #2456
+
+- securejoin: display error reason if there is any #2470
+
+- do not allow deleting contacts with ongoing chats #2458
+
+- ignore drafts folder when scanning #2454
```

Comment:
Idk if that makes a difference, but this PR was mainly a bugfix for https://github.com/deltachat/deltachat-core-rust/issues/2369, although the name doesn't indicate that. We may just want to also prefix it with `fix:`?

Reply:
makes sense, i added a prefix.